crates/oxc_linter/src/rules/react/jsx_props_no_spread_multi.rs

```diff
@@ -0,0 +1,100 @@
+use std::collections::HashMap;
```

Comment:
prefer `FxHashMap` from `rustc_hash`

Reply:
I’ve changed it to FxHashMap.
https://github.com/oxc-project/oxc/pull/4866/commits/af2c3560b566d162346094c2b3970429dc6f52f0